LICENSE

```diff
@@ -220,11 +220,11 @@ For details, see https://github.com/vert-x3/vertx-web
 ================================================================
 For swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/extend/property/AbstractBaseIntegerProperty.java
 ================================================================
-This product bundles files from swagger which is licensed under the Apache License v2.
+This product bundles files from hystrix which is licensed under the Apache License v2.
 For details, see https://github.com/swagger-api/swagger-core
 
 ================================================================
 For handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/HystrixCommandPropertiesExt.java
 ================================================================
 This product bundles files from swagger which is licensed under the Apache License v2.
```

Comment:
Maybe change here?

Reply:
We need to change the line of 229 instead of 223.